File: java/CFCTRL2.java
import java.math.BigInteger;
import java.util.Scanner;

public class CFCTRL2 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int z = sc.nextInt();
        //int count = 1;
        BigInteger count = new BigInteger("1");
        if (z >= 1 && z <= 100) {
            while (z > 0) {
                int a = sc.nextInt();
                if (a >= 1 && a <= 100) {
                    while (a > 0) {
                        count = count.multiply(BigInteger.valueOf(a));
                        a--;
                    }
                    System.out.println(count);
                    count = BigInteger.valueOf(1);
                    z--;
                }
            }
        }
    }
}

File: c++/CFCTRL2.cc
#include <iostream>
using namespace std;

int main()
{
    int z, x;
    cin >> z;
    long long count=1;
    if (z >= 1 && z <= 100)
    {
        while (z > 0)
        {
            cin >> x;
            if (x >= 1 && x <= 100)
            {
                while (x > 0)
                {
                    count = count * x;
                    x = x - 1;
                }
            }
            cout << count << endl;
            count = 1;
            z = z - 1;
        }
    }
}